models/articlesModel.js

```diff
@@ -84,5 +84,16 @@ const fetchArticleComments = (articleId) => {
 		});
 };
 
-module.exports = { fetchArticles, fetchArticleById, updateArticleVotes, fetchArticleComments, addComment };
+const removeComment = (commentId) => {
+	return db.query("DELETE FROM comments WHERE comment_id = $1", [commentId]).then(({rowCount}) => {
```

Comment:
you would need the RETURNING* constraint in your query if you wanted to check if we had actually check if something has been deleted from the database. As right now, I believe rowCount would always be empty, regardless of whether we had deleted a valid comment or not

Reply:
no i think it works, when nothing is deleted rowcount is 0 but when something is the rowcount is the count of rows that were deleted but will check again.